src/test/java/com/ncc/aif/NistExamplesAndValidationTest.java

```diff
@@ -60,13 +60,16 @@ void setup() {
 
         @BeforeEach
         void setup() {
-            ImmutablePair<Resource, Resource> aPair = utils.makeValidNistEntity(SeedlingOntology.Person);
+            ImmutablePair<Resource, Resource> aPair = utils.makeValidNistEntity(
+                    LDCOntology.PER);
```

Comment:
I think breaking these lines up to two or more make sense when the `LDCOntology` constant is really long, but in cases like this, couldn't they just stay on one line?  Not a big deal and I'm certainly fine if you want to keep them all this way for consistency or something.

Or maybe you are using an 80-column width standard?  From what I've seen, the project uses 120 column width.

Reply:
My thinking is that keeping the constants on their own line as much as possible just makes it easy to read and maintain as we move from one ontology to the next.  We are already making changes to specific types based on this review alone - there may be other reasons to change types in the future.